Assets/Scripts/Steam/SteamManager.cs

```diff
@@ -126,8 +129,7 @@ private void OnApplicationQuit()
             }
         }
 
-
-
+        TaskCompletionSource<string> tcs;
```

Comment:
This should be moved to near the definition of the callback. and tcs should be renamed to something like authTicketForWebAPIResponseTaskCompletionSource or something shorter if you can think of it.